.github/workflows/reusable-build.yml

```
@@ -0,0 +1,450 @@
+  # Allow commenting on issues
+  issues: write
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ fromJSON(inputs.runner) }}
+    defaults:
+      run:
+        shell: bash
+    outputs:
+      runner-labels: ${{ steps.upload-artifact.outputs.runner-labels || inputs.runner }}
```

actionlint (1.7.4)

property "runner-labels" is not defined in object type {}

(expression)